### ISSUE TITLE
axe extension reports that aria-label cannot be used on a password input

### ISSUE THREAD

**Issue by fstrr** [dequelabs/axe-core#3234]
Product: axe-core 4.2.0-canary.43394bc running on Chrome.

Expectation:
axe reports an error when an `aria-label` attribute is used on an `<input type="password">` element. [`<input type="password">` allows global aria-* attributes](https://w3c.github.io/html-aria/#el-input-password) and [`aria-label` is listed as one of those](https://www.w3.org/TR/wai-aria-1.1/#global_states) so, unless I'm missing something, this is a bug.

Actual:
axe flags this with "Elements must only use allowed ARIA attributes" and "to solve this violation, you need to fix the following: "ARIA attribute cannot be used: aria-label".

Code:
```html
<input type="password" aria-label="enter your password">
```

Motivation:
axe should report on invalid code and, based on the referenced W3C documents, using `aria-label` on a password appears to be valid.

<pre><code>
axe-core version: .2.0-canary.43394bc
Browser and Assistive Technology versions: Chrome 94.0.4606.81, macOS 10.15.7
</code></pre>
